src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/test

```diff
@@ -0,0 +1,17 @@
+##
+## This test starts a sandbox, deploys a LagomJava service and executes an HTTP request
+## through the Service Gateway to ensure the service is reachable.
+##
+
```

Comment:
Added brief description since this tests differs from others and wanted to clearly state the purpose of the test.